optimum/exporters/onnx/__main__.py

```diff
@@ -143,18 +144,7 @@ def main():
     # Saving the model config as this is needed sometimes.
     model.config.save_pretrained(args.output.parent)
 
-    # Saving the tokenizer / feature extractor as well.
-    try:
-        tokenizer = AutoTokenizer.from_pretrained(args.model)
-        tokenizer.save_pretrained(args.output.parent)
-    except Exception:
-        pass
-
-    try:
-        feature_extractor = AutoFeatureExtractor.from_pretrained(args.model)
-        feature_extractor.save_pretrained(args.output.parent)
-    except Exception:
-        pass
+    maybe_save_tokenizer_or_processor_or_feature_extractor(args.model, args.output.parent)
```

Comment:
I remember discussing to move this inside export, is that still the case?

Reply:
I actually do not know if we should put this inside the export itself, good question.